uddl_test/uddl_test_formatting/universal-data-definition-language-formatter.test.ts

```diff
@@ -46,14 +43,84 @@ describe("Universal Data Definition Language Formatter", () => {
         
         };
         
+    }
+    
+}`,
+    });
+  });
+
+  it("should format cdm with conceptual entity", async () => {
+    await universalDataDefinitionLanguageFormatting({
+      before:
+        'dm PPT "Base data structures to support People, Places and Things"{cdm Conceptual "Need to start at Conceptual Level"{centity AddressableEntity "Any entity that is addressable in some way" {};}}',
+      after: `dm PPT "Base data structures to support People, Places and Things"
+{
+
+    cdm Conceptual "Need to start at Conceptual Level"
+    {
+    
         centity AddressableEntity "Any entity that is addressable in some way"
         {
         
         };
         
     }
     
-}`
+}`,
     });
   });
-});
\ No newline at end of file
+
+  it('should format cdm with conceptual basis',async () => {
```

Comment:
While you can have tests for each individual type, you may find that a test that combines multiple types is sufficient.

Reply:
In a previous commit, I merged conceptual entities and conceptual associations into one test case. During this process, only formatter for conceptual entities was checked. I then attempted to verify whether I would encounter similar behavior for conceptual basis and conceptual domain as well.